### PR TITLE
Make buffer pos more visible in alternate modeline

### PR DESCRIPTION
**Pull request by DamienCassou**

### CODE REVIEW

File: zerodark-theme.el [hunk 1]
```diff
@@ -87,7 +91,7 @@
 (defvar zerodark-modeline-position ":%l:%c %p "
   "Mode line construct for displaying the position in the buffer.")
 
-(defvar zerodark-modeline-position-alt (propertize ":%l:%c %p " 'face 'mode-line-inactive)
+(defvar zerodark-modeline-position-alt (propertize ":%l:%c %p " 'face 'zerodark-buffer-position-alt-face)
   "Alternate mode line construct for displaying the position in the buffer.")
```
Comment (NicolasPetton) [inline zerodark-theme.el:95]: It has to look different in inactive-mode-line.

File: zerodark-theme.el [hunk 2]
```diff
@@ -63,6 +63,10 @@
   '((t :foreground "#98be65" :height 0.9))
   "Face for not modified buffers in the mode-line.")
 
+(defface zerodark-buffer-position-alt-face
+  '((t :foreground "#cccccc" :height 0.9))
```
Comment (NicolasPetton) [inline zerodark-theme.el:67]: `#cccccc` is the same color as the active-mode-line, isn't it? If that's what you want, then we can just remove the face property.

If it's not what you want, you could choose a slightly darker grey to dim it a bit.

**Review by NicolasPetton — CHANGES_REQUESTED** [review]

**Comment by NicolasPetton** [comment]
@DamienCassou could you make future PRs here instead https://petton.fr/git/nico/zerodark-theme?

**Comment by DamienCassou** [comment]
I opened a new one at: https://petton.fr/git/nico/zerodark-theme/pulls/2

**Pull request closed without merging**